stream-to-file/src/file/trace/tests/multiple_digitizers_missing_data.rs

```diff
@@ -0,0 +1,175 @@
+use super::*;
+use ndarray::{arr1, arr2};
+use std::fs;
+use streaming_types::frame_metadata_v1_generated::GpsTime;
+
+#[test]
+fn test_multiple_digitizers_missing_data() {
+    let num_digitizers = 3;
+    let num_time_points = 20;
+    let num_channels = num_digitizers * CHANNELS_PER_DIGITIZER;
+    let num_frames = 3;
+    let num_measurements = num_frames * num_time_points;
+
+    let filepath = create_test_filename("TraceFile_test_multiple_digitizers_missing_data");
+    let mut file = TraceFile::create(&filepath, num_digitizers).unwrap();
+    let _ = fs::remove_file(filepath);
+
+    push_frame(
+        &mut file,
+        num_time_points,
+        0,
+        GpsTime::new(22, 205, 10, 55, 30, 0, 0, 0),
+        0,
+        0,
+    );
+
+    push_frame(
+        &mut file,
+        num_time_points,
+        1,
+        GpsTime::new(22, 205, 10, 55, 30, 20, 0, 0),
+        0,
+        0,
+    );
+
+    push_frame(
+        &mut file,
+        num_time_points,
+        2,
+        GpsTime::new(22, 205, 10, 55, 30, 40, 0, 0),
+        0,
+        0,
+    );
+
+    push_frame(
+        &mut file,
+        num_time_points,
+        0,
+        GpsTime::new(22, 205, 10, 55, 30, 0, 0, 0),
+        0,
+        1,
+    );
+
+    // push_frame(
```

Comment:
Could we delete (or implement) these commented out tests?

Reply:
I'm inclined to leave them as they are, the test data in this test is a copy of the normal case multiple detectors test but with some missing data, this is the missing data.

Reply:
Could you comment as to why this is a comment and not actually used in the test?